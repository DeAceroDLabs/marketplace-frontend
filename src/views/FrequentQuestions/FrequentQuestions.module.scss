.last-question{
    margin-bottom: 10%;
}