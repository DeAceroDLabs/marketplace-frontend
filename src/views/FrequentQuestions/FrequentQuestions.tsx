import React, { useContext, useEffect } from "react";
import styles from "./FrequentQuestions.module.scss";
import Section from "components/common/Section";
import ToogleCategories from "components/common/ToogleCategories";

const FrequentQuestions: React.FunctionComponent = () => {
  return (
    <Section title="Preguntas frecuentes">
      <ToogleCategories
        categoryName="Categoría"
        question="Pregunta 1"
        content="Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., li tot Europa usa li sam vocabularium. 
        Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicos directe al desirabilit?
         de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform grammatica, pronunciation e plu sommun paroles.  "
        defaultVisibility={true}
      />

      <ToogleCategories
        categoryName="Categoría"
        question="Pregunta 2"
        content="Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., li tot Europa usa li sam vocabularium. 
        Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicos directe al desirabilit?
         de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform grammatica, pronunciation e plu sommun paroles.  "
      />
      <ToogleCategories
        categoryName="Categoría"
        question="Pregunta 3"
        content="Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., li tot Europa usa li sam vocabularium. 
        Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicos directe al desirabilit?
         de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform grammatica, pronunciation e plu sommun paroles.  "
      />
      <div className={styles["last-question"]}>
      <ToogleCategories
        categoryName="Categoría"
        question="Pregunta 4"
        content="Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., li tot Europa usa li sam vocabularium. 
        Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicos directe al desirabilit?
         de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform grammatica, pronunciation e plu sommun paroles.  "
      />
      </div>
    </Section>
  );
};

export default FrequentQuestions;
