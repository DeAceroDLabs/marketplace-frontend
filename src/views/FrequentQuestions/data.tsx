const data = [
    {
      "title": "Categoría",
      "question": "Pregunta 1",
      "content": "Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., \
      li tom Europa usa li sam vocabularium. Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicosdirecte \
      al desirabilit? de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform \
      grammatica, pronunciation e plu sommun paroles.",
      "isOpen": true,
    },
    {
      "title": "Categoría",
      "question": "Pregunta 2",
      "content": "Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., \
      li tom Europa usa li sam vocabularium. Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicosdirecte \
      al desirabilit? de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform \
      grammatica, pronunciation e plu sommun paroles.",
    },
    {
      "title": "Categoría",
      "question": "Pregunta 3",
      "content": "Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., \
      li tom Europa usa li sam vocabularium. Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicosdirecte \
      al desirabilit? de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform \
      grammatica, pronunciation e plu sommun paroles.",
    },
    {
      "title": "Categoría",
      "question": "Pregunta 4",
      "content": "Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., \
      li tom Europa usa li sam vocabularium. Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicosdirecte \
      al desirabilit? de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform \
      grammatica, pronunciation e plu sommun paroles.",
    },
  ];

export default data;