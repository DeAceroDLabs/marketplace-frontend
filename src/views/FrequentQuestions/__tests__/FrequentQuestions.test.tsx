import renderer from "react-test-renderer";
import FrequentQuestions from "../FrequentQuestions";


it("renders FrequentQuestions with no issue", () => {
  const component = renderer.create(
    <FrequentQuestions/>

  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
