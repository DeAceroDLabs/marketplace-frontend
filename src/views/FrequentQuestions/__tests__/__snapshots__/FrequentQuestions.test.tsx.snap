// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FrequentQuestions with no issue 1`] = `
<div
  className="container"
>
  <div
    className="title margin-title"
  >
    Preguntas frecuentes
     
  </div>
  <div
    className="body"
  >
     
    <div
      className="container"
    >
      <div
        className="category-arrow-container"
      >
        <div>
          Categoría
        </div>
        <button
          className="button transparent-black activeClass"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ArrowDropUpIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="m7 14 5-5 5 5z"
            />
          </svg>
        </button>
      </div>
      <hr
        className="line"
      />
      <div
        className="question-content-container"
      >
        <div
          className="question"
        >
          Pregunta 1
        </div>
        <div
          className="content"
        >
          Li Europan lingues es membres del sam familie. Lor separat existentie es un myth. Por scientie, musica, sport etc., li tot Europa usa li sam vocabularium. 
 Li lingues differe solmen in li grammatica, li pronunciation e li plu commun vocabules. Omnicos directe al desirabilit?
 de un nov lingua franca: on refusa continuar payar custosi traductores. It solmen va esser necessi far uniform grammatica, pronunciation e plu sommun paroles.  
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="category-arrow-container"
      >
        <div>
          Categoría
        </div>
        <button
          className="button transparent-black activeClass"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ArrowDropDownIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="m7 10 5 5 5-5z"
            />
          </svg>
        </button>
      </div>
      <hr
        className="line"
      />
    </div>
    <div
      className="container"
    >
      <div
        className="category-arrow-container"
      >
        <div>
          Categoría
        </div>
        <button
          className="button transparent-black activeClass"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ArrowDropDownIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="m7 10 5 5 5-5z"
            />
          </svg>
        </button>
      </div>
      <hr
        className="line"
      />
    </div>
    <div
      className="last-question"
    >
      <div
        className="container"
      >
        <div
          className="category-arrow-container"
        >
          <div>
            Categoría
          </div>
          <button
            className="button transparent-black activeClass"
            onClick={[Function]}
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="m7 10 5 5 5-5z"
              />
            </svg>
          </button>
        </div>
        <hr
          className="line"
        />
      </div>
    </div>
     
  </div>
</div>
`;
